### ISSUE TITLE
False positive with combineLatest (ngrx/avoid-mapping-selectors)

### ISSUE THREAD

**Issue by DaSchTour** [timdeschryver/eslint-plugin-ngrx#174]
## Behavior

When combining store with another observable and map the linter tells me to use selectors

## Code snippet

```ts
combineLatest([this.store.select(selectAuthorizations), this.hasAuthorization$]).pipe(map((val) => !isEmpty(intersection(val))))
```

## I would be willing to submit a PR to fix this issue :heart:

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[x] No

**Comment by timdeschryver** [comment]
@rafaelss95 you can reproduce this if you add the following fixture to the valid cases of `avoid-mapping-selectors`. Are you going to work on this?

```ts
      import { Store, select } from '@ngrx/store'
      @Injectable()
      export class FixtureEffect {
        loginUserSuccess$ = combineLatest([this.store.select(selectAuthorizations), this.hasAuthorization$]).pipe(map((val) => !isEmpty(intersection(val))));
        constructor(private store: Store){}
      }
```

**Comment by rafaelss95** [comment]
Danm, I was testing without import declaration 🤦. Sorry for this! But yes, I could try to solve this issue as well.

**Comment by timdeschryver** [comment]
@rafaelss95 no problem!
Please take your time, I didn't meant to rush you - I just wanted to make sure we didn' work on the same issue.
